minhphan156/Abode
# SJSU-1029, Cancel Reservation Frontend (React & Redux)

Cancellation pop up confirmation on /history page, with text detailing refund policy.
Yes (Confirm cancellation) and No (Cancel cancellation) buttons on pop up.

To Test:
1.	Book a new hotel/find an existing hotel associated with your customerID in (mlab likehome database) bookings Collection. 
e.g. copy the **bookingID** string**"5cb76de65f35ee09fcc8e113"** in 
"_id": {
        "$oid": "5cb76de65f35ee09fcc8e113"
    },
And use it to replace the placeholder on **line 93 of HotelOverview.js**
Note that **"status": 0** before cancellation
2.	Go to **localhost:3000/history**, click on CANCEL POP UP and YES, check mlab bookings Collection to see that **"status": 3** for associated bookingID.
